Module 5/Problem 8/problem8_b.json

```diff
@@ -0,0 +1,89 @@
+Input:
+{
+  "employees": [
+    {
+      "name": "John Doe",
+      "salary": 50000,
+      "department": "Sales"
+    },
+    {
+      "name": "Jane Smith",
+      "salary": 60000
+    },
+    {
+      "name": "Bob Johnson",
+      "department": "IT"
+    }
+  ]
+}
+
+
+Transformation:
+    
+[
+  {
+    "operation": "modify-default-beta",
+    "spec": {
+      "employees": {
+        "*": {
+          "department": "Marketing",
+          "salary": 45000
+        },
+        "[0]": {
+          "bonus": 1000,
+          "tempBonus": 1000
+        },
+        "[1]": {
+          "bonus": 1500,
+          "tempBonus": 1500
+        },
+        "[2]": {
+          "bonus": 900,
+          "tempBonus": 900
```

Comment:
In this, the bonus is related to the department in some way. You can add that approach also.